typing/types.mli

```diff
@@ -331,6 +333,12 @@ and constructor_arguments =
   | Cstr_tuple of type_expr list
   | Cstr_record of label_declaration list
 
+and unboxed_status =
```

Comment:
Yes, or even two different fields in the type_declaration record, but the overhead! Do you think the brevity of the code is worth one more word in the data structure?